content/Watsonx/WatsonxGov/Level-4/203.md

```diff
@@ -0,0 +1,155 @@
+    3. Click the ellipsis button on the far right of the OpenScale instance, and then click open. <br /> ![openscale launch](../images/203/watsonxgov-l4_openscale-launch.webp)
+   > NOTE: If the Open option does not appear, the page may not have finished fully loading. Try waiting for up to a minute or refreshing the page.
+    4. If this is the first time you have opened the watsonx.governance monitoring tool, you will see the **Model evaluation** modal window. Click **Manual setup**. Note that you can do an auto setup using the same credentials and database information as the steps below to fill out the dashboard with some example models, but the process will take up to 20 minutes longer. <br /> ![openscale manual](../images/203/watsonxgov-l4_openscale-manual.webp)
+    5. From the **System setup** screen, click the Pencil icon on the Database tile to edit database information.
+    6. Use the following values for the fields:
+        - Database type: **db2**
+        - Database location: **Other**
+        - Hostname or IP Address: **The value you constructed earlier ending in -db2u**
+        - Database: `BLUDB`
+        - Username: `admin`
+        - Password: **Your Cloud Pak for Data admin password** <br /> ![openscale config](../images/203/watsonxgov-l4_openscale-config.webp)
+    7. Click **Connect**. The monitoring service will attempt to connect to the database using the credentials you supplied. If the connection fails, double-check that you have constructed the hostname correctly, and that you are using the admin username and password that you use to log into the Cloud Pak for Data or watsonx home screen.
+    8. After successfully connecting, set **Schema** to **Auto-create a new schema**, and click save. When the changes have successfully saved, the watsonx.governance monitoring service will be configured and operational. <br /> ![openscale db connect](../images/203/watsonxgov-l4_openscale-db-connect.webp)
+
+4. ### Add the user to the monitoring service
+
+    In order to perform model evaluations, the user you created in previous steps needs to be granted access to the monitoring service.
+
+    1. From the CP4D home screen use the hamburger menu, then **Services** -> **Instances**
+    2. Locate the OpenScale service in the table and open the ellipsis menu, then click on **Manage access**. <br /> ![openscale manage access](../images/203/watsonxgov-l4_openscale-manage-access.webp)
+    3. Click on **Add users**, then select the user you created, and set the role to **Admin**, then click **add**. <br /> ![openscale user add](../images/203/watsonxgov-l4_openscale-user-add.webp) <br />The created user will now have Admin privileges, which will allow them to add machine learning providers. Note that assigning Editor privileges would allow the user to create and remove model subscriptions and configure models for monitoring, but not add the deployment spaces or third-party environments that host the models.
```

Comment:
This screenshot should be updated to show the `complianceofficer` username to be consistent

`then select the user you created` can specify `complianceofficer`